### PR TITLE
Update prod_cron_meta_nightly_pipeline.groovy

### PR DESCRIPTION
**Pull request by Ginxo**
related to https://github.com/kiegroup/kie-jenkins-scripts/pull/1232

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Review by mareknovotny — APPROVED** [review]
lgtm, wasn't similar change already done few days ago?

**Comment by mareknovotny** [comment]
ok, it was for kogito only https://github.com/kiegroup/kie-jenkins-scripts/commit/1ba85f22cbce926f022ad302f0dc64e87a6b713b

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -60,7 +60,7 @@ pipeline{
         stage('trigger KOGITO nightly job ${KOGITO_NEXT_PRODUCT_BRANCH}') {
             steps {
                 build job: 'kogito.nightly/${KOGITO_NEXT_PRODUCT_BRANCH}', propagate: false, wait: true, parameters: [
```
Comment (MarianMacik) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:62]: Isn't the issue also here?
Based on the [latest run](https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/blue/organizations/jenkins/PROD%2Fcron-meta-nightly-pipeline/detail/cron-meta-nightly-pipeline/342/pipeline) it tried to run `kogito.nightly/7.67.x`, which doesn't exist of course. The issue is probably that `KOGITO_NEXT_PRODUCT_BRANCH` is the same as `NEXT_PRODUCT_BRANCH`:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy#L20
which is set to `BRANCH`:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy#L12
which is set to 7.67.x:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy#L25

**Review by MarianMacik — CHANGES_REQUESTED** [review]

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy (hunk 1 above)
Comment (winklerm) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:62]: I am actually fixing this in another PR (in progress) with some refactoring to make the pipeline definition less error-prone. I will post the PR here in a while.

Comment (winklerm) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:62]: https://github.com/kiegroup/kie-jenkins-scripts/pull/1234

**Comment by mareknovotny** [comment]
so could we close this in favor of #1234 ?

**Comment by mareknovotny** [comment]
is this forgotten PR @Ginxo ?

**Comment by Ginxo** [comment]
out of date

**Pull request closed without merging**